Backend-README.md

```diff
@@ -0,0 +1,235 @@
+## Chitter Backend
```

Comment:
This file is from [Chitter API Backend](https://github.com/makersacademy/chitter_api_backend) and is only here for reference (not written by me).